servers/rendering/rendering_device.h

```diff
@@ -397,9 +397,10 @@ class RenderingDevice : public Object {
 		BARRIER_MASK_COMPUTE = 2,
 		BARRIER_MASK_TRANSFER = 4,
 		BARRIER_MASK_NO_BARRIER = 8,
-		BARRIER_MASK_ALL = BARRIER_MASK_RASTER | BARRIER_MASK_COMPUTE | BARRIER_MASK_TRANSFER
 	};
 
+	static const BarrierMask BARRIER_MASK_ALL = BarrierMask(BARRIER_MASK_RASTER | BARRIER_MASK_COMPUTE | BARRIER_MASK_TRANSFER);
```

Comment:
I don't like having this outside the enum definition. Everywhere else in the engine we keep the aggregate enums inside the enum definition. 

I understand the awkwardness of having a ``BARRIER_MASK_ALL`` that only includes 3 of the 4 values in the enum. But I think the better solution is to rename it to ``BARRIER_MASK_ALL_BARRIERS``. because it is the barrier mask representing all barriers, rather than the enum representing all barrier masks. Does that make sense?

Reply:
Make sense, done.